ERCS/erc-7678.md

```diff
@@ -0,0 +1,223 @@
+---
+eip: 7678
```

Comment:
```suggestion
eip: 7679
```
Numbers are assigned by ERC editors/associates. Please don't self assign. Number sniping is not allowed.
Manually assigning next sequential number.


Please also update the filename.

Reply:
apologies, my mistake! i was just trying [wrongly, I think,] to resolve a Walidator error about the filename

Reply:
No worries.  To avoid any perception of number sniping I manually bump self assigners numbers by 1.